docs/products/kafka/howto/flink-with-aiven-for-kafka.rst

```diff
@@ -0,0 +1,169 @@
+Use Apache Flink® with Aiven for Apache Kafka®
+======================================================
+
+`Apache Flink® <https://flink.apache.org/>`_ is an open source platform for processing distributed streaming and batch data.
+Where Apache Kafka® excels at receiving and sending event streams, Flink consumes, transforms, aggregates, and enriches your data.
+
+The example in this article shows you how to create a simple Java Flink job that reads data from a Kafka topic, processes it, and then pushes it to a different Kafka topic.
+
+.. Note::
+
+   If you want to experience the power of streaming SQL transformations with Flink, Aiven provides a hosted :doc:`/docs/products/flink` with built-in data flow integration with Aiven for Apache Kafka®.
+
+   This example demonstrates using the Java API on a `local installation of Apache Flink 1.15.1 <https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/try-flink/local_installation/>`_, but it can be applied to use Aiven for Apache Kafka with any self-hosted cluster.
+
+.. _kafka-flink-prereq:
+
+Prerequisites
+-------------
+
+You need an Aiven for Apache Kafka service up and running with two topics, named ``test-flink-input`` and ``test-flink-output`` already created. Furthermore, for the example, you need to collect the following information about the Aiven for Apache Kafka service:
```

Comment:
The topic creation could be automated in the example, although requiring the user to use Aiven Console or Client makes our services more familiar. Should a link to Aiven for Apache Kafka instructions be here, https://developer.aiven.io/docs/products/kafka/howto/create-topic

Reply:
:+1:  I think I prefer to refer to the docs to keep the document focused!  (Also, setting `auto.create.topics.enable` is probably not a great practice for users copy/pasting this code).

Reply:
I did not specifically recommend the `auto.create.topics.enable`, but having the example to create topic e.g. Kafka admin client.

Reply:
Oh, I just meant that to point out that the "one-liner" way of doing the autocreate is probably not great!  So I'll go with the link (which is my preference when doing code tutorials).